### ISSUE TITLE
[2.6] Whiteboard component crash

### ISSUE THREAD

**Issue by schrd** [bigbluebutton/bigbluebutton#16153]
**Describe the bug**
During the community test today I had a crash of the whiteboard component. The stack trace I got was this one:

~~~
TypeError: this.pageState is undefined
    get selectedIds https://bbbserver/html5client/ac539f824232acc920f6ecdba0c1a1758f51d485.js?meteor_js_resource=true:9
    p3 https://bbbserver/html5client/ac539f824232acc920f6ecdba0c1a1758f51d485.js?meteor_js_resource=true:9
    p3 https://bbbserver/html5client/ac539f824232acc920f6ecdba0c1a1758f51d485.js?meteor_js_resource=true:9
    Y0 https://bbbserver/html5client/ac539f824232acc920f6ecdba0c1a1758f51d485.js?meteor_js_resource=true:9
    p3 https://bbbserver/html5client/ac539f824232acc920f6ecdba0c1a1758f51d485.js?meteor_js_resource=true:9
    T https://bbbserver/html5client/ac539f824232acc920f6ecdba0c1a1758f51d485.js?meteor_js_resource=true:432
    du https://bbbserver/html5client/ac539f824232acc920f6ecdba0c1a1758f51d485.js?meteor_js_resource=true:61
    unstable_runWithPriority https://bbbserver/html5client/ac539f824232acc920f6ecdba0c1a1758f51d485.js?meteor_js_resource=true:79
    bo https://bbbserver/html5client/ac539f824232acc920f6ecdba0c1a1758f51d485.js?meteor_js_resource=true:61
    lu https://bbbserver/html5client/ac539f824232acc920f6ecdba0c1a1758f51d485.js?meteor_js_resource=true:61
    su https://bbbserver/html5client/ac539f824232acc920f6ecdba0c1a1758f51d485.js?meteor_js_resource=true:61
    A https://bbbserver/html5client/ac539f824232acc920f6ecdba0c1a1758f51d485.js?meteor_js_resource=true:79
    onmessage https://bbbserver/html5client/ac539f824232acc920f6ecdba0c1a1758f51d485.js?meteor_js_resource=true:79
react_devtools_backend.js:4012:25

~~~

**To Reproduce**
Unfortunately I could not reproduce it and I was the only participant in the meeting with this crash.

**BBB version:**
2.6 beta, commit ID 84e6ba2d408bf364d428c8e3c7a044cbeab7228b

**Desktop (please complete the following information):**
 - OS: Linux
 - Browser Firefox
 - Version 102

**Comment by antobinary** [comment]
Thanks for reporting!
These are the only spots I find `pageState` https://github.com/bigbluebutton/bigbluebutton/blob/v2.6.x-release/bigbluebutton-html5/imports/ui/components/whiteboard/component.jsx#L501-L537
Possibly we need an early exit if `e` is undefined on ~line 502.

**Comment by ramonlsouza** [comment]
fixed by #16569
